routes/categories-api.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express');
+const router = express.Router();
+
+const categoryQueries = require('../db/queries/categories');
+const itemsForCategoryQueries = require('../db/queries/items');
+
+// Get all categories from a user
+router.get('/', (req, res) => {
+
+  const userId = req.cookies.user_id;
+
+  if (!req.cookies || !req.cookies.user_id) {
+    return res.status(401).json({ message: 'Please sign up or log in to create lists.' });
+  }
+
+  categoryQueries
+    .getCategoriesForUser(userId)
+    .then((categories) => {
+      res.json({ categories });
```

Comment:
Maybe we can use `send` to return the object instead of the json, since it avoids having to parse it on the client.